host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1265,4 +1266,12 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @LogMessage(level = Level.ERROR)
     @Message( id = 172, value = "Failed executing the suspend operation for server: %s")
     void suspendExecutionFailed(@Cause IOException cause, String serverName);
+
+    @LogMessage(level = Level.INFO)
```

Comment:
Yes, for it makes sense an INFO, I think :), I can argue two reasons:
1) Only the changes in this issue are using these log traces and if those exceptions are threw the returning operation is still a success (there are no other message for the user, he will see success as a result of :suspend-servers if a timeout occurrs) and maybe the suspend in one server was correct. For this reason I think that maybe it shouldn't be an error.
2) I followed the same traces in org.jboss.as.domain.controller.logging.DomainControllerLogger, there same traces are logged as INFO instead of ERROR.

Well, maybe should they be an error if the suspend operation failed for all existing servers? and even more, shouldn't the returning operation be an error instead of a success in that case?
Just let me know how do you see it.